documentation/document.html

```diff
@@ -556,6 +556,70 @@ <h1 class="comp-title">Rate Product</h1>
+                  <ul class="flex-col center-it list-container">
+                    <li class="flex-row center-it spc-btwn norm-list-item">
+                      <p>The quantity of mangoes:</p>
+                      <div>18</div>
+                    </li>
+                    <li class="flex-row center-it spc-btwn norm-list-item">
+                      <p>The quantity of bananas:</p>
+                      <div>10</div>
+                    </li>
+                    <li class="flex-row center-it spc-btwn norm-list-item">
+                      <p>The quantity of apples :</p>
+                      <div>12</div>
+                    </li>
+                  </ul>
+                  <iframe src="https://www.thiscodeworks.com/embed/6212bc0f11c4db00151a7dc4" style="width: 100%; height: 268px;" frameborder="0"></iframe>
+                  <!-- STACKED LIST -->
+                  <h3>STACKED LIST</h3>
+                  <ul class="flex-col center-it stk-list-container">
+                    <li class="flex-row center-it gap-btwn stack-list-item">
+                      <div class="flex-col gap-btwn">
+                        <span class="material-icons"> feed </span>1 hour ago
+                      </div>
+                      <div class="flex-col gap-btwn">
+                        Top story
+                        <p class="stack-list-text">
+                          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        </p>
+                      </div>
+                      <img src="../ignore/modi.jfif" alt="" class="stk-list-img" />
+                    </li>
+                    <li class="flex-row center-it gap-btwn stack-list-item">
+                      <div class="flex-col gap-btwn">
+                        <span class="material-icons"> feed </span>1 hour ago
+                      </div>
+                      <div class="flex-col gap-btwn">
+                        Top story
+                        <p class="stack-list-text">
+                          Est perspiciatis beatae facilis voluptatem iste?
+                          Necessitatibus corrupti minus
+                        </p>
+                      </div>
+                      <img src="../ignore/river.jfif" alt="" class="stk-list-img" />
+                    </li>
+                    <li class="flex-row center-it gap-btwn stack-list-item">
+                      <div class="flex-col gap-btwn">
+                        <span class="material-icons"> feed </span>1 hour ago
+                      </div>
+                      <div class="flex-col gap-btwn">
+                        Top story
+                        <p class="stack-list-text">
+                          nulla blanditiis, quia, numquam nihil quo facilis, amet id
+                          recusandae? Nihil, cum aspernatur?
+                        </p>
+                      </div>
+                      <img src="../ignore/cricket.jfif" alt="" class="stk-list-img" />
```

Comment:
Just a suggestion that you can use class name  as stack-list-img rather than stk-list-img